packages/patterns/src/components/reset-overrides-control.js

```diff
@@ -6,80 +6,52 @@ import {
 	BlockControls,
 } from '@wordpress/block-editor';
 import { ToolbarButton, ToolbarGroup } from '@wordpress/components';
-import { useSelect, useRegistry } from '@wordpress/data';
-import { store as coreStore } from '@wordpress/core-data';
-import { parse } from '@wordpress/blocks';
+import { useDispatch, useSelect } from '@wordpress/data';
 import { __ } from '@wordpress/i18n';
 
-function recursivelyFindBlockWithName( blocks, name ) {
-	for ( const block of blocks ) {
-		if ( block.attributes.metadata?.name === name ) {
-			return block;
-		}
-
-		const found = recursivelyFindBlockWithName( block.innerBlocks, name );
-		if ( found ) {
-			return found;
-		}
-	}
-}
+const CONTENT = 'content';
 
 export default function ResetOverridesControl( props ) {
-	const registry = useRegistry();
 	const name = props.attributes.metadata?.name;
-	const patternWithOverrides = useSelect(
+	const { updateBlockAttributes } = useDispatch( blockEditorStore );
+
+	const { isOverwritten, resetOverrides } = useSelect(
```

Comment:
`isOverridden` would be a better name IMO.

Reply:
Totally right. Changed in [this commit](https://github.com/WordPress/gutenberg/pull/60721/commits/501ade3c978108dccc87f4e6b94124772351b6b7).